zulipterminal/ui_tools/boxes.py

```diff
@@ -402,15 +402,29 @@ def main_view(self) -> List[Any]:
         soup = BeautifulSoup(self.message['content'], 'lxml')
         content = (None, self.soup2markup(soup.find(name='body')))
         active_char = '▒'  # Options are '█', '▓', '▒', '░'
+
+        message_content = dict(
```

Comment:
`message_content` seems like a strange name for this? You only have two entries now - perhaps just have them called that?

You could also include this as a branch in the conditional below - they are set to one of two values, dependent on that.

Reply:
sure. done